serverless/serverless
# Issue #5502 Support for native async/await in AWS Lambda for aws-nodejs-typescript template

## What did you implement:

Closes #5502 



## How did you implement it:
Changed the plugins/create/templates/aws-nodejs-typescript/tsconfig.json to contain es2017 for lib and target



## How can we verify it:
Run _serverless create --template aws-nodejs-typescript_ which should now target es2017 instead of es6



## Todos:

- [n/a] Write tests
- [n/a] Write documentation
- [n/a] Fix linting errors
- [n/a] Make sure code coverage hasn't dropped
- [n/a] Provide verification config / commands / resources

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
